RELEASE/scripts/autoscend/auto_equipment.ash

```diff
@@ -700,14 +700,19 @@ void finalizeMaximize(boolean speculative)
 		// blocks first hit, but doesn't stack with blood bubble
 		addBonusToMaximize($item[Eight Days a Week Pill Keeper], 100);
 	}
-	if (possessEquipment($item[June cleaver])) {
-		addBonusToMaximize($item[June cleaver], 100); // We want to ramp this up and the NCs are nice as well
+	
+	if (canUseCleaver()) {
+		if(get_property("_juneCleaverFightsLeft").to_int() < my_adventures()*1.1 && (fullness_limit() > 0 || inebriety_limit() > 0) && consumptionProgress() >= 1) {
```

Comment:
was giving an example of when not to give a maximizer bonus, so rewriting the same as a condition for when to give bonus it would become like
>if(get_property("_juneCleaverFightsLeft").to_int() < my_adventures()*1.1 || 
>!((fullness_limit() > 0 || inebriety_limit() > 0) && consumptionProgress() >= 1))

where the second line condition means turngen may not be over, forgot to explain in example that (fullness_limit() > 0 || inebriety_limit() > 0 check is just because consumptionProgress() gives no information about special turngen for classes that don't have either of these organs